### DISCUSSION TITLE
Tips on how to migrate from valet / homebrew / nvm / etc.

### DISCUSSION THREAD

**Discussion by marcamos** [beyondcode/herd-community#1156] [Q&A]
Hey everybody, I'm finally looking at Herd as a _simpler_ way to do what I've been doing for years with Valet, Homebrew, NVM, etc. and I'm wondering if there's anything I should do _before_ I install Herd to ensure I don't create any overlap headaches for myself?

For example:
* I use Homebrew / Valet and have 20+ different sites, with varying versions of PHP, SSL, etc.
* I use NVM with _at least_ five different versions of Node on-hand
* I use DBngin for databases (and SequelAce to move them around)
* Some of these things start up when my mac boots up … but I'm fuzzy on exactly _what_ does this 🤔 

…so, yeah, should I be _uninstalling_ anything first in order to make the transition smooth?

**Comment by whoacowboy** [comment]
@marcamos Thinking of doing the same, did you learn anything worthwhile? Best, james

**Reply by marcamos** [reply]
Hi @whoacowboy, since I wrote the original post I got a new computer which gave me a blank slate to start with.

I took a day to start with Herd from scratch, added a few sites to it (which came with its own [question](https://github.com/beyondcode/herd-community/discussions/1207)), and have been up and running ever since.

So, in the end, I never needed to do the migration I originally asked about.

**Reply by whoacowboy** [reply]
Hey @marcamos, thanks for the response and I am glad it is working. I imagine it's not that difficult. I am loath the mess with my dev environment, but valet is starting to give me issues on restart so I just decided to make the switch. 
Best

**Reply by marcamos** [reply]
You got it; best of luck!
